pkg/quota/generic/evaluator.go

```diff
@@ -41,6 +41,9 @@ func ListResourceUsingInformerFunc(f informers.SharedInformerFactory, resource s
 		if err != nil {
 			return nil, err
 		}
+		if !informer.Informer().HasSynced() {
```

Comment:
Yes, The scenario is I created a rs and has 30 replicas, but the quota is just 10. 
So there will be only 10 pod that can be created. But after I restart all of the kube processes on the master node, I find that the  created pod increased to 20, then I restart  again, the pod number increased to 30. I watched the quota during the startup, the result shows  the used pod becomes to 0  during start period.  I can  reproduce this every time with 1.5, but can't reproduce with 1.6.   Anyway I think it's better to check this, even there are no issues left.